README.md

```diff
@@ -8,3 +8,26 @@ Utilities for maintaining the [Fedora Commons repository](http://github.com/fcre
     * fedora:mimeType => ebucore:hasMimeType
     * premis:hasOriginalName => ebucore:filename
 
+Building
+--------
+
+To build the JAR file
+
+``` sh
+mvn package
+```
+
+Running
+-------
+
+To run the migration utility, execute the JAR file and provide the `fcrepo.home` system property to set the directory that contains Fedora 4's `fcrepo4-data` directory.
```

Comment:
Should the existing repository be stopped? Should it be suggested that no other users be interacting with the repo while running this utility?